docs/book/src/design/ledger-platform.md

```diff
@@ -0,0 +1,32 @@
+# Ledger platform
+
+Nautilus Wallet's design supports interfacing with a choice of underlying ledger platform(s).
+
+The system currently supports the following ledgers and asset types:
+
+## Algorand
+
+The [Algorand] ledger supports transacting using [Algorand Standard Assets (ASAs)][ASA].
+
+The Wallet TEE uses an SGX port of the [algonaut] SDK to implement key generation and transaction signing.
+
+The Wallet Connector uses [js-algorand-sdk] to communicate with the Algorand network.
+
+[Algorand]: https://www.algorand.com/
+[ASA]: (https://developer.algorand.org/docs/get-details/asa/)
```

Comment:
Link broken.  Perhaps it should be the following:
```suggestion
[ASA]: https://developer.algorand.org/docs/get-details/asa/
```

Reply:
Good catch, thanks!